Makefile.nt

```diff
@@ -373,7 +373,8 @@ otherlibs/dynlink/dynlink.cmxa: otherlibs/dynlink/natdynlink.ml
 
 utils/config.ml: utils/config.mlp config/Makefile
 	@rm -f utils/config.ml
-	sed -e "s|%%LIBDIR%%|$(LIBDIR)|" \
+	# Convert forward-slashes in LIBDIR to backslashes
+	sed -e "s|%%LIBDIR%%|$(shell echo $(LIBDIR)|sed -e 's:/:\\\\\\\\\\\\\\\\:g')|" \
```

Comment:
Doh - seeing the wood for the trees! Can't believe I didn't think to use `$(subst ..)` instead! I'll push a new version later.